### ISSUE TITLE
Ds erros

### ISSUE THREAD

**Issue by gui565br** [rafaelvcaetano/melonDS-android#196]
* **Device model:** moto g(6) plus
* **Android version:** Android API 28
* **melonDS version:** Beta 1.3.0

**Problem:**  
Insert a small description of the problem.

**Stack trace:**  
```
me.magnum.melonds.ui.emulator.EmulatorActivity$d: Failed to load firmware: BIOS9_MISSING
	at me.magnum.melonds.ui.emulator.h$c.a(Unknown Source:67)
	at f.a.b0.e.a.a.g(Unknown Source:10)
	at f.a.a.a(Unknown Source:14)
	at f.a.b0.e.a.e$a.run(Unknown Source:2)
	at f.a.r$a.run(Unknown Source:9)
	at f.a.b0.g.k.run(Unknown Source:13)
	at f.a.b0.g.k.call(Unknown Source:0)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:764)

```

**Configuration:**  
```
FirmwareCrashContext(emulatorConfiguration=EmulatorConfiguration(useCustomBios=true, dsConfigDirectory=//, dsiConfigDirectory=//, internalDirectory=/data/user/0/me.magnum.melonds/files, fastForwardSpeedMultiplier=-1.0, useJit=true, consoleType=DS, soundEnabled=true, micSource=NONE, firmwareConfiguration=FirmwareConfiguration(nickname=Player, message=Hello!, language=1, favouriteColour=0, birthdayMonth=1, birthdayDay=1), rendererConfiguration=RendererConfiguration(videoFiltering=NONE, threadedRendering=false)), dsiBiosDirUri=content://com.android.providers.downloads.documents/tree/raw%3A%2Fstorage%2Femulated%2F0%2FDownload%2FBios%20Melonds%2FBios%20NDS, dsBiosDirUri=content://com.android.providers.downloads.documents/tree/raw%3A%2Fstorage%2Femulated%2F0%2FDownload%2FBios%20Melonds%2FBios%20NDSi)
```

**Comment by rafaelvcaetano** [comment]
Try to put your BIOS files outside the Download directory.

**Comment by rafaelvcaetano** [comment]
If your problem is not yet fixed, try the latest [Beta 1.4.0](https://github.com/rafaelvcaetano/melonDS-android/releases/latest) release. If the problem is not fixed, please open a new issue.
